README.md

```diff
@@ -1,9 +1,50 @@
-[![Build Status](https://travis-ci.org/TREP-LABS/realdrip-backend.svg?branch=master)](https://travis-ci.org/TREP-LABS/realdrip-backend)
-[![Coverage Status](https://coveralls.io/repos/github/TREP-LABS/realdrip-backend/badge.svg?branch=master)](https://coveralls.io/github/TREP-LABS/realdrip-backend?branch=master)
-## RealDrip Backend
-`A node-js server using express-js
-`
-#### The Setup integrated the following CI/CD tools
- - TravisCI
- - Coveralls.io
- - Jest
\ No newline at end of file
+# RealDrip Backend [![Build Status](https://travis-ci.org/TREP-LABS/realdrip-backend.svg?branch=master)](https://travis-ci.org/TREP-LABS/realdrip-backend)  [![Coverage Status](https://coveralls.io/repos/github/TREP-LABS/realdrip-backend/badge.svg?branch=master)](https://coveralls.io/github/TREP-LABS/realdrip-backend?branch=master)
+
+> What is RealDrip? Check [here](https://treplabs.co/realdrip)
+
+This is the backend API of the RealDrip platform.
+
+## API Documentation
+The API is well documented using the OpenAPI Spec(Swagger) [here](https://app.swaggerhub.com/apis/Treplabs/Realdrip_Platform/1.0#/).
+
+## Getting Started
+The instructions below would get you a copy of the project up and running on your local machine
+
+### Prerequisite
+- Git
+- Node
+- npm or yarn
+
+### Installation
+- Clone the repository `git clone https://github.com/TREP-LABS/realdrip-backend.git`
+- Change into the directory of the project
+- Use `npm install` to install all the dependency packages
+- Create a `.env` file in the root folder of the to provide all the needed environment variables as specified in `.env.example`
+- Use `npm run dev` to start the application in dev mode
+
+## Continuous Integration(CI)
+We try our best to continuously integrate our changes together as soon as they are created i.e we don't do long live feature branches. All pull requests are raised directly unto the shared main branch(master). You can read more about continuous integrations [here](https://en.wikipedia.org/wiki/Continuous_integration). We use a third-party platform to effectively manage our continuous integration, [TravisCI](https://travis-ci.org/).Travis helps us run automated builds for every commit introduced into the codebase. At any point in time, if Travis reports a failing build, fixing such a build should be the priority of the team. For CI, Travis builds includes linting the codebase to ensure it adheres to the standard of the team and running automated tests to ensure nothing is breaking.
+
+## Continuous Delivery(CD)
+We also practice Continuous Delivery, every change introduced(merged) into the master branch is automatically deployed to a staging(pre-production) environment where it can be tested as it would be in production. You can read more about Continuous Delivery[here](https://en.wikipedia.org/wiki/Continuous_delivery);
```

Comment:
```suggestion
We also practice Continuous Delivery, every change introduced (merged) into the master branch is automatically deployed to a staging (pre-production) environment where it can be tested as it would be in production. You can read more about Continuous Delivery [here](https://en.wikipedia.org/wiki/Continuous_delivery);
```